tests/ttnn/unit_tests/operations/test_paged_update_cache.py

```diff
@@ -65,30 +86,51 @@ def run_test_update_cache_decode(
         eq_update, output_update = comp_pcc(x, tt_updated_slice)  # checks the updated parts
     logger.debug(output_cache)
     logger.debug(output_update)
+
+    if (not eq_cache or not eq_update) and logger.level("DEBUG"):
+        # find deltas between cache and tt_got_back
+        cache_delta = cache - tt_got_back
+        for i in range(max_seq_len):
+            if cache_dtype == ttnn.bfloat16:
+                if not torch.sum(cache_delta[:, :, i, :]) == 0:
+                    logger.error(f"cache_delta at {i}: {cache_delta[:, :, i, :]}")
+                    logger.info(f"cache at {i}: {cache[:, :, i, :]}")
+                    logger.info(f"tt_got_back at {i}: {tt_got_back[:, :, i, :]}")
+            else:
+                eq, pcc = comp_pcc(cache[:, :, i, :], tt_got_back[:, :, i, :])
+                if not eq:
+                    logger.error(f"cache_delta {pcc} pcc at {i}: {cache_delta[:, :, i, :]}")
+                    logger.info(f"cache at {i}: {cache[:, :, i, :]}")
+                    logger.info(f"tt_got_back at {i}: {tt_got_back[:, :, i, :]}")
+
     assert eq_cache and eq_update
 
 
-@pytest.mark.skip("Test case covered by others")
+@pytest.mark.skipif(is_grayskull(), reason="Grayskull does not support paged cache")
 @pytest.mark.parametrize("check_memory", [False])
+@pytest.mark.parametrize("share_cache", [True, False])
 @pytest.mark.parametrize("head_dim", [128])
-@pytest.mark.parametrize("max_seq_len", [2048])
-@pytest.mark.parametrize("num_users", [32])
-@pytest.mark.parametrize("num_heads", [1])
-@pytest.mark.parametrize("input_dtype", [ttnn.bfloat16, ttnn.bfloat8_b])
-@pytest.mark.parametrize("cache_idx", [0, 1, 127, 1057])
-@pytest.mark.parametrize("cache_dtype", [ttnn.bfloat16, ttnn.bfloat8_b])
+@pytest.mark.parametrize("max_seq_len", [2048, 32 * 1024])
+@pytest.mark.parametrize("num_users", [4])
+@pytest.mark.parametrize("num_heads", [8])
```

Comment:
Test parameters have changed. Do we not care about the old cases anymore?

Reply:
This particular one is okay as it was covered by other test cases